sklearn/preprocessing/data.py

```diff
@@ -1629,18 +1633,27 @@ def _transform_selected(X, transform, selected="all", copy=True):
     transform : callable
         A callable transform(X) -> X_transformed
 
-    copy : boolean, optional
+    copy : boolean, default=True
         Copy X even if it could be avoided.
 
-    selected: "all" or array of indices or mask
+    selected : "all" or array of indices or mask
         Specify which features to apply the transform to.
 
+    retain_order : boolean, default=False
+        If True, the non-selected features will not be displaced to the right
+        side of the transformed array. The number of features in Xt must
```

Comment:
Technically, the non-selected features aren't reinserted into `Xt`; they're just not modified by the transformation.